app/models/program.rb

```diff
@@ -94,4 +95,15 @@ def create_difficulty_levels
                                                 color: '#EF6E69')
     true
   end
+
+  ##
+  # Check if languages string has the right format. Used as before_save.
+  #
+  def check_languages_format
+    # All white spaces are removed to allow languages to be separated by ',' and ', ' but only the first option is saved
+    if self.languages.present?
+      self.languages = self.languages.delete(' ')
+      self.languages.match(/^$|(\A[a-z][a-z](,[a-z][a-z])*\z)/).present?
```

Comment:
No, as the regular expression checks that it is a list of words of two letters. Although you could enter `fo` as language and it is not. I could split the string and check if every language is a real language.